packages/documentation/src/stories/welcome.stories.mdx

```diff
@@ -12,14 +15,203 @@ import { Meta } from '@storybook/addon-docs';
   }}
 />
 
-<div
-  style={{
-    margin: '2rem 0',
-    padding: '1rem',
-    background: 'red',
-    fontWeight: 'bold',
-    color: 'white',
-  }}
->
-  TODO: write welcome page
+<div className="mt-huge text-center">
+  <h2 className="h3 mb-big">Packages</h2>
+
+<p>We offer different packages, which can all be used together or individually.</p>
+
+<div className="mt-regular d-flex gap-big flex-wrap justify-content-center">
+  <Tile title="Styles" href="https://github.com/swisspost/design-system/tree/main/packages/styles">
+    <img src="/images/packages/styles.svg" alt="design-system-styles package logo" />
+    <span>v{getVersion('styles')}</span>
+  </Tile>
+  <Tile
+    title="Components"
+    href="https://github.com/swisspost/design-system/tree/main/packages/components"
+  >
+    <img src="/images/packages/components.svg" alt="design-system-components package Logo" />
+    <span>v{getVersion('components')}</span>
+  </Tile>
+  <Tile
+    title="Internet-Header"
+    href="https://github.com/swisspost/design-system/tree/main/packages/internet-header"
+  >
+    <img src="/images/packages/internet-header.svg" alt="internet-header package Logo" />
+    <span>v{getVersion('internetheader')}</span>
+  </Tile>
+  <Tile
+    title="Intranet-Header"
+    href="https://github.com/swisspost/design-system/tree/main/packages/components-angular/projects/intranet-header"
+  >
+    <img
+      src="/images/packages/intranet-header.svg"
+      alt="design-system-intranet-header package Logo"
+    />
+    <span>v{getVersion('intranetheader')}</span>
+  </Tile>
+  <Tile title="Icons" href="https://github.com/swisspost/design-system/tree/main/packages/icons">
+    <img src="/images/packages/icons.svg" alt="design-system-icons package Logo" />
+    <span>v{getVersion('icons')}</span>
+  </Tile>
+</div>
+
+<div className="mt-huge mb-regular d-flex gap-mini flex-wrap justify-content-center">
+  {/* <div className="w-100 mb-regular">
+    <a href="#" className="btn btn-primary btn-lg">
+      Get started
+    </a>
+  </div> */}
```

Comment:
Add a todo page "Getting started" and include the primary button again or remove the commented out code.